packages/round-manager/src/features/program/CreateProgramPage.tsx

```diff
@@ -117,6 +120,38 @@ export default function CreateProgram() {
           : ProgressStatus.NOT_STARTED,
     },
   ];
+
+  function ProgramChain() {
+    return (
+      <>
+        <InformationCircleIcon
+          data-tip
+          data-background-color="#0E0333"
+          data-for="program-chain-tooltip"
+          className="inline h-3 w-3 ml-1 mb-1"
+          data-testid={"program-chain-tooltip"}
+        />
+
+        <ReactTooltip
+          id="program-chain-tooltip"
+          place="bottom"
+          type="dark"
+          effect="solid"
+        >
+          <p className="text-xs">
+            All associated grant rounds will need to <br />
+            run on the chain to which you deploy your <br />
+            program. If you want to use a different <br />
+            chain after deployment you will need to <br />
+            create a new program. To change the <br />
+            chain for your program, use the network <br />
+            selection tool in your nav bar.
```

Comment:
We should avoid using names like "nav bar" and instead use conventional terminology - i.e. "navigation bar"